src/components/License/VLicenseElements.vue

```diff
@@ -23,21 +23,26 @@
 <script>
 import { computed, defineComponent } from '@nuxtjs/composition-api'
 
+import { getElements } from '~/utils/license'
+
 import VIcon from '~/components/VIcon/VIcon.vue'
 
 import by from '~/assets/licenses/by.svg'
-import cc0 from '~/assets/licenses/cc0.svg'
+import zero from '~/assets/licenses/zero.svg'
 import nc from '~/assets/licenses/nc.svg'
 import nd from '~/assets/licenses/nd.svg'
-import pdm from '~/assets/licenses/pdm.svg'
+import pd from '~/assets/licenses/pd.svg'
 import sa from '~/assets/licenses/sa.svg'
-import sampling from '~/assets/licenses/sampling.svg'
-import samplingPlus from '~/assets/licenses/sampling-plus.svg'
+import samplingPlus from '~/assets/licenses/sampling.plus.svg'
 
 export default defineComponent({
   name: 'VLicenseElements',
   components: { VIcon },
   props: {
+    /**
+     * the slug of the license
+     * @values
```

Comment:
Is this jsdoc incomplete?

Reply:
Yeah, had to specify the options after `@values`.